luke-lukemichaeleisenberg/hw_2/4-fizzbuzz-homework/app.js

```diff
@@ -0,0 +1,34 @@
+
+// Relying on your newfound knowledge of loops, combine loops and if/else statements together and incrementally build the common fizzbuzz loop.
+
+// - In the loop, every time a number is divisible by **3**, instead of logging the number itself, the word "fizz" should appear.
+// - If the number is divisible by  **5**, the word "buzz" should be logged.
+// - If the number is divisible by both **3** and  **5**, then the word "fizzbuzz" should be logged.
+
+// Follow the steps below.
+
+// ##### Step 1:
+// Construct a for loop that iterates through, and `console.log()`'s out, numbers 1 - 100.
+
+// ##### Step 2:
+// Add an `if/else` statement that logs the string `"fizz"` if the value being iterated over is divisible by `3`; otherwise, log out the value.
+
+// ##### Step 3:
+// Add an `else if` clause that logs the string `"buzz"` if the value being iterated over is divisible by `5`.
+
+// ##### Step 4:
+// Add an additional `else if` clause that logs the string `"fizzbuzz"` if the value being iterated over is divisible by both `3` and `5`.
+
+for (var i = 1 ; i <= 100; i++)
+{
```

Comment:
https://github.com/airbnb/javascript#the-javascript-style-guide-guide

```
A function foo ()
B {
C     print "Hello";
D     print "World";
E }
```
This is misleading because you can't tell if line B is subordinate to line A or if line B is its own separate statement. Here's a quick explanation of why the curly bracket is expected to be included on the first line along with the control statement:

Functions have two main elements:
* the control construct
* the function's statements

By using the curly bracket on the second line (line B in the image) it is neither in the control statement nor is it part of the function's statements.

```
function foo() { // brace is on the same line and it's associated with the statement that it opens
    print "Hello"; // this line does something
    print "World"; // this line does something
} // this last brace terminates the function which is part of the control construct
```